### PR TITLE
Removal of Windows msi installer related function

### PR DESCRIPTION
**Pull request by marbx**
### What does this PR do?
- Removes code related to the Windows msi installer builder.
  - The builder containes the removed code since [PR #41](https://github.com/saltstack/salt-windows-msi/pull/41)

### Tests written?
No, only code removed.

### CODE REVIEW

**Comment by markuskramerIgitt** [comment]
@twangboy could you please pick up and review?

**Comment by markuskramerIgitt** [comment]
### ci/py3/windows2016  error 

Traceback (most recent call last):
  File "c:\users\admini~1\appdata\local\temp\kitchen\testing\tests\integration\modules\test_mine.py", line 258, in test_mine_delete
    self.assertDictEqual(ret_args, expected)
AssertionError: {} != {'minion': {'args': [], 'kwargs': {'foo': 'bar', 'fnord': 'roscivs'}}}
- {}
+ {'minion': {'args': [], 'kwargs': {'fnord': 'roscivs', 'foo': 'bar'}}}

### ci/py2/macosxmojave error

Traceback (most recent call last):
  File "/private/tmp/kitchen/testing/tests/support/helpers.py", line 1023, in <lambda>
    lambda testcase, *args, **kwargs: self.wrap(testcase, *args, **kwargs)  # pylint: disable=W0108
  File "/private/tmp/kitchen/testing/tests/support/helpers.py", line 1031, in wrap
    return self.func(testcase, name, *args, **kwargs)
  File "/private/tmp/kitchen/testing/tests/integration/modules/test_cp.py", line 402, in test_get_url_ftp
    self.assertIn('Base system', data)
AssertionError: u'Base system' not found in u''

File: salt/version.py [hunk 1]
```diff
@@ -773,38 +777,39 @@ def versions_report(include_salt_cloud=False):
...
-      2018.8          2018.3.2-1306        18.42.1306
-      v2016.11.0      2016.11.0            16.200.65535        Tags have noc 0
-      v2016.11.2      2016.11.2            16.202.65535
+    The "display version" is free format.
 
+    Examples:
+      Git-tag                msi display version         msi internal version
+      v3000                  3000                        30.00.0.0
+      v3001                  3001                        30.01.0.0
+      v3001.1                3000.1                      30.01.1.0
```
Comment (twangboy) [inline salt/version.py]: msi display version needs to match the git-tag: `3001.1`

**Review by twangboy — CHANGES_REQUESTED** [review]

File: salt/version.py (hunk 1 above)
Comment (markuskramerIgitt) [reply salt/version.py]: Thanks. Corrected.

Comment (markuskramerIgitt) [reply salt/version.py]: @twangboy, I moved the mis internal version routine to salt-windows-msi PowerShell. 
I think this is the much better place.

**Review by twangboy — DISMISSED** [review]

**Review by Ch3LL — DISMISSED** [review]

**Comment by Ch3LL** [comment]
ping @markuskramerIgitt looks like there is a conflict here that needs to be resolved. im guessing this conflict occurred with migrating to using the black tool in this PR addition: https://github.com/saltstack/salt/pull/55765

**Comment by markuskramerIgitt** [comment]
Thank you @Ch3LL 
Conflict solved

**Comment by markuskramerIgitt** [comment]
Why does the ci/pre-commit check fails?

**Comment by DmitryKuzmenko** [comment]
@markuskramerIgitt we are using `isor` and `black` for code formatting. You can follow the `Details` link on the pre-commit check and then click `Console Output` and scroll down to `black ...... Failed`. It shows the diff with changes recommended to be applied.
Also it's recommended to install pre-commit hook that would do the job for you before commit.
```
pip install pre-commit
pre-commit install
```
After this pre-commit would run automatically on commit or you can run it manually for your changes using `pre-commit run --files $(git show --pretty="" --name-only master..HEAD)` or just run isort and black on your changes with this:
```
FILES="$(git show --pretty="" --name-only master..HEAD)"
pre-commit run isort --files $FILES
pre-commit run black --files $FILES
```

**Comment by markuskramerIgitt** [comment]
Thank you @DmitryKuzmenko 
ci/pre-commit passed

**Comment by dwoz** [comment]
@twangboy Does this look good to you now?

**Review by Ch3LL — APPROVED** [review]

**Review by twangboy — APPROVED** [review]

**Comment by twangboy** [comment]
@dwoz Yeah, looks good.

**Pull request merged**